cdap-docs/admin-manual/source/operations/preferences.rst

```diff
@@ -76,4 +76,33 @@ or provide Twitter API credentials::
       queue = new LinkedBlockingQueue<Tweet>(10000);
       collector = new CollectingThread();
       collector.start();
-    }
\ No newline at end of file
+    }
+
+Scoped Runtime Arguments for Workflow
+=====================================
+When a Workflow is configured, you may want to pass specific runtime arguments to the different programs
+and datasets used inside the Workflow. To achieve this, you can prefix the runtime arguments with a ``<scope>.``.
+Currently supported scopes are ``dataset``, ``mapreduce``, and ``spark``.
+
+Example: To set a runtime argument of ``read.timeout=30`` for the MapReduce program ``oneMapReduce`` in the Workflow,
+the argument can be provided with a scope of ``mapreduce.oneMapReduce.read.timeout=30``. In this case, ``oneMapReduce``
+and the datasets used in ``oneMapReduce`` will receive two arguments: one with a scope of
+``mapreduce.oneMapReduce.read.timeout=30``, and another with the scope extracted as ``read.timeout=30``.
+Programs other than ``oneMapReduce`` and datasets used in them will receive only the single argument
+``mapreduce.oneMapReduce.read.timeout=30``.
+
+An argument can also be prefixed with ``<scope>.*`` to apply it to all programs or datasets in the Workflow as
+represented by the scope.
+
+Example: If the runtime argument is specified as ``mapreduce.*.read.timeout=30``, all MapReduce programs and
+datasets used in them will receive two arguments - one with the scope ``mapreduce.*.read.timeout=30`` and another
+with the scope extracted ``read.timeout=30``. Programs other than the MapReduce and datasets used them
```

Comment:
Programs other than the MapReduce and the datasets used by them \[is this correct?]